src/gateways/Gateway.php

```diff
@@ -41,17 +40,17 @@ class Gateway extends BaseGateway
 	use GatewayTrait;
 
 	//Settings options
-	public $api_key;
-	public $private_key;
-	public $analyticsId;
-	public $brandingId;
-	public $autoCapture;
-	public $autoCaptureStatus;
-	public $enableAutoStatus;
-	public $afterCaptureStatus;
-	public $paymentMethods;
-	public $enabled3ds;
-	public $convertAmount;
+	public string $api_key = '';
+	public string $private_key = '';
+	public string $analyticsId = '';
+	public string $brandingId = '';
+	public int $autoCapture = 0;
+	public bool $autoCaptureStatus = false;
+	public string $enableAutoStatus = '';
```

Comment:
Er ikke en string, men en boolean
```suggestion
	public boolean $enableAutoStatus = false;
```